druntime/src/core/internal/gc/bits.d

```diff
@@ -50,10 +53,13 @@ struct GCBits
     void alloc(size_t nbits, bool share = false) nothrow
     {
         this.nbits = nbits;
-        static if (!HaveFork)
-            data = cast(typeof(data[0])*)calloc(nwords, data[0].sizeof);
-        else if (share)
-            data = cast(typeof(data[0])*)os_mem_map(nwords * data[0].sizeof, true); // Allocate as MAP_SHARED
+        static if (AllocSupportsShared)
```

Comment:
Instead of magic constant that one has to remember setting, this could instead be
```suggestion
        static if (__traits(compile, os_mem_map))
```
or
```suggestion
        static if (__traits(hasMember, core.internal.gc.os, "os_mem_map"))
```